lib/src/result.dart

```diff
@@ -37,7 +37,7 @@ abstract class Result<T> {
   /// The result of the transformation is a sink that only forwards [Result]
   /// values and no error events.
   static const StreamSinkTransformer<Object, Result> captureSinkTransformer =
-      const StreamSinkTransformer.fromStreamTransformer(
```

Comment:
I think because `const CaptureStreamTransformer()` is a `StreamTransformer<dynamic, Result>` and not a `StreamTransformer<Object, Result>`

I *think* we could fix this instead with `const CaptureStreamTransformer<Object>()`